htdocs/reception/class/reception.class.php

```diff
@@ -378,7 +378,7 @@ public function fetch($id, $ref = '', $ref_ext = '')
 		$sql .= ', e.fk_incoterms, e.location_incoterms';
 		$sql .= ', i.libelle as label_incoterms';
 		$sql .= " FROM ".MAIN_DB_PREFIX."reception as e";
-		$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."element_element as el ON el.fk_target = e.rowid AND el.targettype = '".$this->db->escape($this->element)."'";
+		$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."element_element as el ON el.fk_target = e.rowid AND el.targettype = '".$this->db->escape($this->element)."' AND el.sourcetype = order_supplier";
```

Comment:
`= 'order_supplier'"` ?

Reply:
Pardon my french but : “M’enfin!”
I cleaned my glasses, it should go better 😒